packages/keyv-redis/src/index.js

```diff
@@ -82,7 +77,7 @@ class KeyvRedis extends EventEmitter {
 			}
 		}
 
-		yield * iterate(0, `${this.namespace}:*`);
+		yield * iterate(0, `${this.namespace}*`);
```

Comment:
this potentially should be reverted.

Can you check https://github.com/keyvhq/keyv/pull/9/files#r642115526 ?

Reply:
Suppose a user is using the same redis with no namespace and with a valid namespace, won't clearing the one without a namespace delete entries which do have a valid namespace?

Reply:
yes, that's part of the trade-off, if you use don't setup a namespace and call `.clear`, it's going to delete all the things.

Reply:
Aight then, we need to make sure all other stores follow this behaviour as well to ensure consistency